php7-intro/index.md

```diff
@@ -237,16 +247,16 @@ Here we will get an ```E_COMPILE_ERROR``` cause of broken *invariance*.
 
 ## Anonymous Classes
 
-A concept of anonymous class is already applied in a quite a range of programming languages from Java to Ruby. Since PHP is mostly used for an object oriented programming(we just forget about Wordpress here) anonymous classes definitely make sense to apply it for simple *pass and forget* objects in use cases like testing and talking to libraries in general.
+The concept of anonymous classes are already available in quite a range of programming languages from Java to Ruby. Since PHP now a days is mostly used for object oriented programming (We just need to forget about Wordpress here ;)), anonymous classes definitely make sense in for simple *pass and forget* objects. Also use cases like testing and talking to libraries in general can make use of anonymous classes.
```

Comment:
is that doesn't  have to be:

 "The concept of anonymous classes **is**..." 

 "nowadays"

 "anonymous classes definitely make sense for simple"